docs/user_guide/overview.rst

```diff
@@ -117,6 +119,28 @@ participants in project.yml) specified when generating the startup kits in the p
 correct IP. If the FL server is on an internal network without a DNS hostname, in Ubuntu, an entry may need to be added
```

Comment:
This part needs to be renamed to Overseer?

Reply:
forgot to change after copy paste